cedar-policy/src/api.rs

```diff
@@ -3137,6 +3154,43 @@ impl Request {
     ///
     /// If `schema` is present, this constructor will validate that the
     /// `Request` complies with the given `schema`.
```

Comment:
```suggestion
    /// `Request` complies with the given `schema`.
    /// 
    /// This function is provided to provide backwards-compatible behavior with
    /// Cedar version 3.x. It will be removed in a future release.
```